src/components/shapeStackUtils.js

```diff
@@ -44,6 +46,12 @@ function bringShapeToFront(shape) {
   shape.toFront({ deep: true });
 }
 
+function bringFlowsToFront(graph) {
+  if (graph) {
```

Comment:
Could it ever be possible that graph is falsey here? If not let's remove the if check.